ssh/ssher.py

```diff
@@ -91,6 +91,7 @@ def open_tunnel(user: str, key: str, host: str, port: int=22) -> Tunnelled:
             '-oControlPath=' + temp_paths[0],
             '-oStrictHostKeyChecking=no',
             '-oUserKnownHostsFile=/dev/null',
+            '-oLogLevel=ERROR',
```

Comment:
This is my understanding as well. In general, I think its OK to stray on the side of less logging information here because this client is intended for high-level application only. That being said, we will still get all critical information in the event of a considerable issue